lib/app_web/controllers/google_auth_controller.ex

```diff
@@ -1,7 +1,8 @@
 defmodule AppWeb.GoogleAuthController do
   use AppWeb, :controller
 
-  @elixir_auth_google Application.get_env(:app, :elixir_auth_google) || ElixirAuthGoogle
+  @elixir_auth_google Application.get_env(:app, :elixir_auth_google) ||
+                        ElixirAuthGoogle
```

Comment:
I don't understand why `mix format` is indenting this `ElixirAuthGoogle` by 24 characters ... 😕 
Not that it's a _problem_, I just want to _understand_ if it's consistent. 💭